utility/utility_test.bzl

```diff
@@ -0,0 +1,18 @@
+load("//:oppia_android_test.bzl", "oppia_android_test")
+
+def utility_test(name, src, test_class):
+  '''
+  This macro exists as a way to customize the oppia_android_test() macro for the utility module.
```

Comment:
This documentation should describe what the macro does & how to use it, not how it's implemented (devs can read the code to find that out). Please also document the arguments to provide context on how to use them.

Reply:
Okay!